src/components/src/main/java/org/apache/jmeter/visualizers/PropertyControlGui.java

```diff
@@ -133,18 +131,15 @@ private void setUpData(){
         }
         Set<Map.Entry<Object, Object>> s = p.entrySet();
         List<Map.Entry<Object, Object>> al = new ArrayList<>(s);
-        Collections.sort(al, new Comparator<Map.Entry<Object, Object>>(){
-            @Override
-            public int compare(Map.Entry<Object, Object> o1, Map.Entry<Object, Object> o2) {
-                String m1 = (String) o1.getKey();
-                String m2 = (String) o2.getKey();
-                return m1.compareTo(m2);
-            }
+        al.sort((o1, o2) -> {
```

Comment:
I guess this can be further simplified by using  `Comparator.comparing(Map.Entry::getKey)`?

Reply:
That doesn't work out of the box for me, as the Map isn't typed strong enough.
```
al.sort(Comparator.comparing(o -> (String) o.getKey()));
```
could be good, though.

Reply:
That doesn't seem to compile, I think they need to be cast to Strings first.